innovation_sweet_spots/tests/data_update_checks/utils.py

```diff
@@ -0,0 +1,193 @@
+from innovation_sweet_spots import PROJECT_DIR, logger
+import pathlib
+import pandas as pd
+import seaborn as sns
+from innovation_sweet_spots.utils.io import load_json
+
+sns.set_theme(style="whitegrid")
+
+DUC_DIR = PROJECT_DIR / "innovation_sweet_spots/tests/data_update_checks/"
+
+
+def load_file_as_df(dir_path: pathlib.Path, filename: str) -> pd.DataFrame:
+    """Load file as dataframe without any processing"""
+    path = dir_path / filename
+    if path.suffix == ".csv":
+        return pd.read_csv(path)
+    if path.suffix == ".json":
+        return pd.DataFrame(load_json(path))
+
+
+def find_filenames(
+    path: pathlib.Path,
+) -> list:
+    """Return a set of filenames from path"""
+    return {file.name for file in path.iterdir() if file.name != ".DS_Store"}
+
+
+def check_filenames_match(cur_files: set, new_files: set, cur_dir: str, new_dir: str):
+    """Check that current and new sets of filenames are the same.
+    Log the outcome and provide additional information on the
+    differences."""
+    logger.info("Checking filenames match...")
+    if cur_files == new_files:
+        logger.info(f"{cur_dir}/ and {new_dir}/ files match.")
+    else:
+        logger.warning(f"{cur_dir}/ and {new_dir}/ files do not match.")
+        if missing_files := cur_files - new_files:
```

Comment:
Hey, this line is throwing a syntax error for me:

```
Traceback (most recent call last):
  File "innovation_sweet_spots/tests/data_update_checks/data_update_checks.py", line 1, in <module>
    from innovation_sweet_spots.tests.data_update_checks.utils import (
  File "/Users/karliskanders/Documents/innovation_sweet_spots/innovation_sweet_spots/tests/data_update_checks/utils.py", line 37
    if missing_files := cur_files - new_files:
                      ^
SyntaxError: invalid syntax
```

Reply:
Probably because I'm still in a Python 3.7 environment. I wonder if there is an equivalent syntax that still works for 3.7? (otherwise I can also upgrade my environment)